TESTS.md

```diff
@@ -10,6 +10,16 @@ particularly useful for checking that errors are printed.
 
 Tip: use `v -cc tcc` when compiling tests for speed.
 
+## Run only single test file
+
+In order to test only one file, use `v test` command
```

Comment:
That is not correct.
`v test` is designed to be used for one or many _test.v files. 
It produces a summary for all of them, no matter how many you give it.
It also searches recursively in all the folders you give it too for _test.v files, so you can do `v test .` and have it run all the local tests in the current folder.

If someone wants to run a single `x_test.v` file - use just `v x_test.v` .
For statistics about the individual test functions in that file - use `v -stats x_test.v` .

Reply:
You are right, my explanations were not correct. What I wanted to say is that we can use `v test` on a single test file.
It was just not explicit in the doc.